### PR TITLE
bump versions

### PR DESCRIPTION
**Pull request by mattrobenolt**
setuptools 20.0
pip 8.0.2
virtualenv 14.0.6
virtualenvwrapper 4.7.1

### CODE REVIEW

**Comment by brainsik** [comment]
Please bump setuptools to 20.0 to match what virtualenv 14.0.6 uses. Thanks!

**Comment by mattrobenolt** [comment]
doh, one minute. :)

**Comment by mattrobenolt** [comment]
@brainsik Done.

File: test_download.py [hunk 1]
```diff
@@ -10,10 +10,10 @@
 
 PYPI_DOWNLOADS = {
     # filename: md5sum
-    'setuptools-18.2.tar.gz': '52b4e48939ef311d7204f8fe940764f4',
-    'pip-7.1.2.tar.gz': '3823d2343d9f3aaab21cf9c917710196',
-    'virtualenv-13.1.2.tar.gz': 'b989598f068d64b32dead530eb25589a',
-    'virtualenvwrapper-4.6.0.tar.gz': 'b6928707eb17152b6a14fd6eeb2931a3',
+    'setuptools-20.1.1.tar.gz': '10a0f4feb9f2ea99acf634c8d7136d6d',
```
Comment (brainsik) [inline test_download.py]: Please use version 20.0 to match exactly what virtualenv 14.0.6 uses.

Comment (mattrobenolt) [inline test_download.py]: Oh, I see. virtualenv packages up specific versions already: https://github.com/pypa/virtualenv/tree/14.0.6/virtualenv_support

Comment (brainsik) [inline test_download.py]: Exactly. Things work best when the versions stay aligned.

**Comment by mattrobenolt** [comment]
@brainsik setuptools 20.0

**Pull request merged**